src/css/global.scss

```diff
@@ -160,7 +169,12 @@ img {
       margin: 0;
     }
   }
+
+  strong{
+    color: $orange;
+  }
 }
+
 .candidate-list {
   font-size: 125%;
```

Comment:
I would recommend using 'em' here instead. It will allow for more consistency across browsers. 

Supporting documents: 
http://stackoverflow.com/questions/132685/font-size-in-css-or-em
http://alistapart.com/article/howtosizetextincss